migration/migrating-3-4/planning-migration-3-to-4.adoc

```diff
@@ -12,7 +12,7 @@ Before performing your migration to {product-title} 4.2, it is important to take
 This planning document assumes that you are transitioning from {product-title} 3.11 to {product-title} 4.2.
 ====
 
-This document provides high-level information on the most important xref:../migration/planning-migration-3-to-4.adoc#migration-comparing-ocp-3-4[differences between {product-title} 3 and {product-title} 4] and the most noteworthy xref:../migration/planning-migration-3-to-4.adoc#migration-considerations[migration considerations]. For detailed information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For detailed information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-2-release-notes.adoc#ocp-4-2-release-notes[OpenShift Container Platform 4.2 release notes].
+This document provides high-level information on the most important xref:../../migration/planning-migration-3-to-4.adoc#migration-comparing-ocp-3-4[differences between {product-title} 3 and {product-title} 4] and the most noteworthy xref:../migration/planning-migration-3-to-4.adoc#migration-considerations[migration considerations]. For detailed information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For detailed information on new features and other notable technical changes, review the xref:../../release_notes/ocp-4-2-release-notes.adoc#ocp-4-2-release-notes[OpenShift Container Platform 4.2 release notes].
```

Comment:
@apinnick, the cherry-pick bot failed because of a conflict in this paragraph. I picked the version that had a link to the 4.2 release notes.